google/ggrc-core
# GGRC-1785 Rename titles of fields in New Task/Edit Tasks modal window

# Issue description

Currently Tree view column names on My Tasks tab correspond with Task info pane labels. The following changes were made:
1. Title -> Task title
2. Workflow -> Cycle title
3. State -> Task state
4. Assignee -> Task assignee
5. Start date -> Task start date
6. End date -> Task due date
7. Last updated -> Task last updated
Update such fields in New Task/Edit Task modal window

# Steps to test the changes

1. Login to GGRC
2. Go to My Tasks tab and click on Edit object on any task. Check if titles of fields on Edit Task Modal window look like below
3. Click Create task button and check if titles of fields on New Task Modal window look like below
    Title -> Task Title
    Active Workflow -> Cycle Title
    Start Date -> Task Start Date
    End Date -> Task Due Date

# Solution description

Modify field titles in src/ggrc-client/js/templates/cycle_task_group_object_tasks/modal_content.stache file

# Sanity checklist




# PR Review checklist
